source/localizable/_code-of-conduct.en.md

```diff
@@ -1,39 +1,39 @@
 Purpose
 -------
 
-A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+A primary goal of all the conferences and user groups that follow this Code of Conduct is to create an inclusive community that welcomes and supports as many people with various backgrounds and identities as possible. As such, we are committed to build a friendly and safer environment for all. These various backgrounds and identities include (but are not limited to) gender identity and expression, sex, sexual orientation, educational level, mental and physical ability, immigration status, race, ethnicity, national origin, social and economic status, age, size and religion (or lack thereof).
 
 This Code of Conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
 
-We invite all those who participate in our events to help us create safe and positive experiences for everyone.
+We invite all those who participate in our events to help us create safer and positive experiences for everyone.
 
 
 Open [Source/Culture/Tech] Citizenship
 --------------------------------------
 
-A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and understand how our actions effect our community.
 
-Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
+Communities mirror the societies in which they exist. Positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
 
-If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+If you see someone making an extra effort to ensure a welcoming, friendly, and encouraging community, we are happy to hear about it.
 
 
 Expected Behavior
 -----------------
 
-*	Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
-*	Exercise consideration and respect in your speech and actions.
+*	Participate in a reliable and sharing way. In doing so, you contribute to the health and longevity of this community.
```

Comment:
I'd prefer active over sharing, as sharing is a subset of active and active covers many more important aspects to me.